src/site/layouts/landing_page.drupal.liquid

```diff
@@ -9,43 +10,46 @@
         <div class="va-introtext">
           <p>{{ fieldIntroText }}</p>
         </div>
-          {% if fieldSpokes != empty %}
-            <h3>On This Page:</h3>
-            <ul>
-              {% for spoke in fieldSpokes %}
-                <li>
-                    <a href="#{% if spoke.entity.fieldTitle != empty %}{{ spoke.entity.fieldTitle | hashReference: title }}{% endif %}">
-                        {{ spoke.entity.fieldTitle }}
-                    </a>
-                </li>
-              {% endfor %}
-            </ul>
-          {% endif %}
+        {% if fieldSpokes != empty %}
```

Comment:
Not a huge deal but is it possible to revert the changes to indentations in this template? It looks like you just meant to include the alert.